client/src/main/java/com/walmartlabs/concord/client/SecretClient.java

```diff
@@ -136,6 +138,55 @@ public String encryptString(UUID instanceId, String orgName, String projectName,
         throw new ApiException("Error encrypting string. Status code:" + r.getStatusCode() + " Data: " + r.getData());
     }
 
+    public SecretOperationResponse createSecret(SecretRequest secretRequest) throws IOException {
+        String path = "/api/v1/org/" + secretRequest.org() + "/secret";
+
+        Map<String, Object> params = new HashMap<>();
+        params.put("name", secretRequest.name());
```

Comment:
AFAIR, concordclient depends on concord-common and therefore has access to `Constants`. Maybe use those constants there?